samples/msal-browser-samples/VanillaJSTestApp2.0/app/customizable-e2e-test/test/browserAAD.spec.ts

```diff
@@ -165,7 +163,7 @@ describe("AAD-PPE Tests", () => {
         beforeEach(async () => {
             context = await browser.createIncognitoBrowserContext();
             page = await context.newPage();
-            page.setDefaultTimeout(ONE_SECOND_IN_MS*5);
+            page.setDefaultTimeout(ONE_SECOND_IN_MS*10);
```

Comment:
Reliability issues on these?  You're making this change everywhere.

Reply:
These were consistently timing out on local. There's a 30000 ms cap for tests defined as default [here ](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/09f65ee73fa849e71e3a73b857f562fab9d47263/samples/msal-browser-samples/VanillaJSTestApp2.0/jest.config.js#L14). Need to consult why this was defined on the individual test level.